Dockerfile.nginx

```diff
@@ -0,0 +1,97 @@
+# Note: This comment was in the previous dockerfile
+# Note: Not 100% sure why
+#
+# Small note, make sure to create upload/thumbs prior to running this! 
+
+#
+# builder container
+FROM docker.io/debian:bookworm-slim AS builder
+
+# install some packages
+RUN set -exu \
+  && DEBIAN_FRONTEND=noninteractive apt-get -yq update \
+  && DEBIAN_FRONTEND=noninteractive apt-get -yq install \
+    libjpeg-dev \
+    libpng-dev \
+    libfreetype6-dev \
+    software-properties-common \
+    ca-certificates \
+    lsb-release \
+    apt-transport-https \
+    curl
+
+# setup nodejs repo
+RUN set -exu \
+  && curl -fsSL https://deb.nodesource.com/gpgkey/nodesource.gpg.key | gpg --dearmor | apt-key add - \
+  && echo "deb https://deb.nodesource.com/node_14.x bookworm main" | tee /etc/apt/sources.list.d/nodesource.list
+
+# acquire install-php-extensions script
+RUN set -exu \
+  && curl -fsSL https://github.com/mlocati/docker-php-extension-installer/releases/download/2.1.28/install-php-extensions \
+    -o /usr/local/bin/install-php-extensions \
+  && chmod +x /usr/local/bin/install-php-extensions
+
+# setup php5.6 repo
+RUN set -exu \
+  && echo "deb https://packages.sury.org/php/ $(lsb_release -sc) main" > /etc/apt/sources.list.d/php.list \
+  && curl -fsSL https://packages.sury.org/php/apt.gpg | apt-key add -
+
+# install php5.6, some extensions, and nodejs
+RUN set -exu \
+  && DEBIAN_FRONTEND=noninteractive apt-get -yq update \
+  && DEBIAN_FRONTEND=noninteractive apt-get -yq install \
+    php5.6 \
+    php5.6-fpm \
+    php5.6-mysqli \
+    php5.6-mysql \
+    php5.6-exif \
+    php5.6-gd \
```

Comment:
If all these php extensions are being installed via apt-get, what is the `docker-php-extension-installer` doing exactly?

Reply:
Nothing, good catch, it shouldn't be in there anymore.